src/Microsoft.DotNet.Darc/src/DarcLib/VirtualMonoRepo/VmrUpdater.cs

```diff
@@ -555,4 +561,12 @@ private record DependencyUpdate(
         string? TargetRevision,
         string? TargetVersion,
         string? Parent);
+
+    private class RepositoryNotInitializedException : Exception
```

Comment:
Why do we have a dedicated exception for this? We don't define any custom data for this exception so would simply throwing an Exception on line 553 work?